statefulsets/kafka/kafka.yaml

```diff
@@ -74,7 +74,7 @@ spec:
         - "exec kafka-server-start.sh /opt/kafka/config/server.properties --override broker.id=${HOSTNAME##*-} \
           --override listeners=PLAINTEXT://:9093 \
           --override zookeeper.connect=zk-0.zk-svc.default.svc.cluster.local:2181,zk-1.zk-svc.default.svc.cluster.local:2181,zk-2.zk-svc.default.svc.cluster.local:2181 \
-          --override log.dir=/var/lib/kafka \
+          --override log.dirs=/var/lib/kafka \
```

Comment:
this should be changed to /var/lib/kafka/data or something different than the base mount in the example here:  https://github.com/kubernetes/contrib/blob/master/statefulsets/kafka/kafka.yaml#L168

this is to avoid the lost+found complaints from kafka.

Reply:
You are probably right, but I guess, that it is more about permissions managing on external volume and pod itself, rather than getting the only proper directory path value in start configuration. Besides, I consider the whole yaml as a template, and complaint is quite straightforward to proper understanding of the issue and exact solution to it.

My PR is more about misleading key, which did mislead me to lose actual data.